components/Agenda/Agenda.tsx

```diff
@@ -75,13 +75,13 @@ export const Agenda: React.FC<AgendaProps> = props => {
   }, [sessions, props.selectedSessionId])
 
   const onSelectHandler = (session: Session, sponsor: Sponsor) => {
-    const url = `/agenda?sessionId=${session.Id}`
+    const url = `${Router.pathname}?sessionId=${session.Id}`
```

Comment:
Changed this so that you can use permalinks to archived agenda sessions too

Reply:
I think we could add the old ones to this new structure but I believe it is a low priority for now.